scripts/e2e-test.sh

```diff
@@ -74,6 +74,10 @@ EOF
 # Parse arguments
 while [ "$#" -gt 0 ]; do
   case "$1" in
+    -m|--mayastor)
```

Comment:
You'll need to add --mayastor to help() above

Reply:
Done.